From 491e90e407394920ae5b24591174ecd81bec5003 Mon Sep 17 00:00:00 2001
From: Giuseppe Scrivano <gscrivan@redhat.com>
Date: Fri, 20 Dec 2013 18:58:33 +0100
Subject: [PATCH 02/10] support: mark SPICE as supported on RHEL-6

RHEL-ONLY

RHEL-6.x quirks are not maintained anymore upstream, so this patch
doesn't make sense there.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>
---
 virtinst/support.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/virtinst/support.py b/virtinst/support.py
index 1fb3e28..2874a90 100644
--- a/virtinst/support.py
+++ b/virtinst/support.py
@@ -242,12 +242,14 @@ _support_dict = {
     SUPPORT_CONN_HV_GRAPHICS_SPICE : {
         "version" : 8006,
         "force_version" : True,
+        "rhel6_drv_version" : [ ("qemu", 12001) ],
         "drv_version" : [ ("qemu", 14000), ],
     },
 
     SUPPORT_CONN_HV_CHAR_SPICEVMC : {
         "version" : 8008,
         "force_version" : True,
+        "rhel6_drv_version" : [ ("qemu", 12001) ],
         "drv_version" : [ ("qemu", 14000), ],
     },
     SUPPORT_CONN_HV_DIRECT_INTERFACE : {
-- 
1.8.5.3

